### PR TITLE
refactor: refactor belongsToUpdater to class

### PR DESCRIPTION
**Pull request by GuillaumeCisco**
## Pull Request checklist:

- [x] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org)
- [x] Create automatic tests
- [x] Test manually the implemented changes
- [x] Review my own code (indentation, syntax, style, simplicity, readability)
- [x] Wonder if you can improve the existing code

### CODE REVIEW

File: src/utils/association-record.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+async function get(model, pk) {
+  const record = await orm.findRecord(model, pk);
+  if (!record) {
+    throw new Error(`related ${model.name} with pk ${pk} does not exist.`);
+  }
+  return record;
+}
+
+exports.get = get;
```
Comment (GuillaumeCisco) [inline src/utils/association-record.js:11]: in another file because will be needed for https://github.com/ForestAdmin/forest-express-sequelize/pull/585

File: src/services/belongs-to-updater.js [hunk 2]
```diff
@@ -1,40 +1,47 @@
...
-            .then((recordAssociated) => {
-              record[setterName](recordAssociated, options);
-            });
-        }
-        return record[setterName](data.data ? data.data.id : null, options);
-      });
-  };
+// WORKAROUND: Make the hasOne associations update work while waiting
+//             for the Sequelize 4 release with the fix of the following
+//             issue: https://github.com/sequelize/sequelize/issues/6069
```
Comment (GuillaumeCisco) [inline src/services/belongs-to-updater.js:7]: Removing this part should be done in another PR.
targetKey will be the id of the record and not the record itself.
It will need intensive testing

**Review by arnaud-moncel — APPROVED** [review]
LGTM, it seem to be better now
